lib/Service/LangRopeService.php

```diff
@@ -228,4 +232,20 @@ public function requestToExApp(
 			];
 		}
 	}
+
+	public function getWithPresentableSources(string $llmResponse, string ...$sourceRefs): string {
+		if (count($sourceRefs) === 0) {
+			return '';
```

Comment:
```suggestion
			return $llmResponse;
```

Should this return the original response instead of the empty string?

Reply:
forgot to change it when changed the function's purpose :p